scripts/packages/fedora/BUILD

```diff
@@ -0,0 +1,23 @@
+genrule(
+    name="bazel-fedora",
+    srcs = [
+        "bazel.spec",
+        "//scripts/packages:bazel",
+        "//scripts/packages:bazel-real",
+        "//scripts/packages:debian/bazel.bazelrc",
```

Comment:
I have another branch in progress to push the debian package generation into //scripts/packages/debian:, so I will move the file as part of that.